plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/TrinoRow.java

```diff
@@ -13,50 +13,39 @@
  */
 package io.trino.plugin.iceberg.delete;
 
-import com.google.common.collect.AbstractIterator;
 import io.trino.spi.Page;
 import io.trino.spi.type.Type;
 import org.apache.iceberg.StructLike;
 
-import javax.annotation.Nullable;
+import java.util.Arrays;
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static io.trino.plugin.iceberg.IcebergPageSink.getIcebergValue;
-import static java.util.Objects.requireNonNull;
 
-public class TrinoRow
+final class TrinoRow
```

Comment:
add `@ThreadSafe`

(i think it's important that this class is thread safe)

Reply:
Why? We only use it in a single place that doesn't require thread safety.